applications/registration-react/src/pages/dataset-registration-page/form-concept/form-concept-pure.component.jsx

```diff
@@ -17,9 +19,9 @@ export const FormConceptPure = ({ languages, errors }) => (
       />
       <Field
         name="concepts"
-        type="text"
-        component={ConceptTagsInputField}
-        label={localization.schema.concept.conceptLabel}
+        component={LookupTagsInputField}
+        getTagFromItem={item => getTranslateText(item.prefLabel)}
+        renderLookupInput={renderConceptAutosuggestForTagsInput}
```

Comment:
Is it possible to move this render lookup function into LookupTagsInputField component? It seems a bit out of place here.

Reply:
this is exaclty the point of the refactoring - there is one generic Lookup component, that will get the detail component of how and what to loookup as a parameter, so different properties will have different choices.

Reply:
I think I will wait with merge, until I have the actual second (and third) use for this component - api lookup for dataset, and dataset lookup for api.